test/test_automl/test_automl.py

```diff
@@ -168,7 +197,7 @@ def test_delete_non_candidate_models(self):
         # Assert at least one model file has been deleted and that there were no
         # deletion errors
         log_file_path = glob.glob(os.path.join(
-            backend_api.temporary_directory, 'AutoML(' + str(seed) + '):*.log'))
+            backend_api.temporary_directory, 'autosklearn.ensemble_builder.log'))
```

Comment:
Would it be possible to have multiple processes write to the same logfile or would that require large amounts of work?

Reply:
Done!